src/docs/product/alerts-notifications/notifications/index.mdx

```diff
@@ -9,48 +9,31 @@ redirect_from:
   - /learn/notifications/
 ---
 
-Aside from Alerts, Sentry has other types of notifications, such as Workflow, Deploy, and Quota notifications. All these notifications are sent through email.
+Sentry users with owner permissions and above can create notifications, change default notification settings, and configure notifications according to a project's needs. Notifications are categorized as [Workflow](#workflow), [Deploy](#deploy), and [Quota](#quota).
```

Comment:
@adhiraj I think this is the correct permissions for notifications. Will you confirm?